src/Features/CSharp/Portable/EditAndContinue/TopSyntaxComparer.cs

```diff
@@ -365,6 +375,16 @@ public override bool ValuesEqual(SyntaxNode left, SyntaxNode right)
             return SyntaxFactory.AreEquivalent(left, right, ignoreChildFunction);
         }
 
+        private static SyntaxNode GetBody(SyntaxNode node)
+        {
+            switch (node)
+            {
+                case BaseMethodDeclarationSyntax baseMethodDeclarationSyntax: return baseMethodDeclarationSyntax.Body ?? (SyntaxNode)baseMethodDeclarationSyntax.ExpressionBody.Expression;
```

Comment:
Thanks! Tons of tests failed due to nulls here. Fixed.

---
In reply to: [133794525](https://github.com/dotnet/roslyn/pull/21573#discussion_r133794525) [](ancestors = 133794525)